support-frontend/assets/helpers/user/user.js

```diff
@@ -163,6 +166,9 @@ const init = (dispatch: Function, actions: UserSetStateActions = defaultUserActi
     });
   } else if (emailFromBrowser) {
```

Comment:
Could this if statement go inside the else so that `dispatch(setStateField(window.guardian.geoip.stateCode));` is not repeated?